appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -669,18 +684,13 @@ public PayaraMicroImpl addDeploymentFile(File file) {
     public PayaraMicroImpl addDeployFromGAV(String GAV) {
         //if (runtime != null) {
         checkNotRunning();
-        if (GAVs == null) {
-            GAVs = new LinkedList<>();
+        if (GAV.split("[,:]").length != 3) {
+            throw new IllegalArgumentException(GAV + " is a not valid maven coordinates");
         }
```

Comment:
This looks like duplication of DeployGAVValidator.

I think it would be useful to only have the code once. Dependency from `impl` to `cmd.options` already exists, so it reasonable to reuse the validators.

Reply:
Same as previous comment